PUBLISHING.rst

```diff
@@ -8,6 +8,7 @@ now, please contact us (our contact info is in the README)
 
 This document explains how trtools maintainers should publish new changes.
 Maintainers should reach consensus before going ahead with publishing changes.
+Note that the publishing to PyPI step below will require Melissa's PyPI login.
```

Comment:
prefer to say "will require appropriate PyPI credentials that are only made available to core maintainers of TRTools."